packages-private/vapor-e2e-test/__tests__/todomvc.spec.ts

```diff
@@ -1,195 +1,182 @@
-import path from 'node:path'
-import {
-  E2E_TIMEOUT,
-  setupPuppeteer,
-} from '../../../packages/vue/__tests__/e2e/e2eUtils'
-import connect from 'connect'
-import sirv from 'sirv'
+import { userEvent } from '@vitest/browser/context'
+import { createVaporApp } from 'vue'
+import App from '../todomvc/App.vue'
+import 'todomvc-app-css/index.css'
+import { E2E_TIMEOUT, css, enterValue } from './e2eUtils'
+
+beforeAll(() => {
+  // MVC relies on local storage, but it persists between reruns in watch mode
+  localStorage.clear()
+})
 
 describe('e2e: todomvc', () => {
-  const {
-    page,
-    click,
-    isVisible,
-    count,
-    text,
-    value,
-    isChecked,
-    isFocused,
-    classList,
-    enterValue,
-    clearValue,
-    timeout,
-  } = setupPuppeteer()
-
-  let server: any
-  const port = '8194'
-  beforeAll(() => {
-    server = connect()
-      .use(sirv(path.resolve(import.meta.dirname, '../dist')))
-      .listen(port)
-    process.on('SIGTERM', () => server && server.close())
+  test('vapor', { timeout: E2E_TIMEOUT }, async () => {
```

Comment:
Putting timeout here makes prettier format the test body with indentation of 2 instead of 4